components/cosmetic_filters/resources/data/cosmetic_filters.js

```diff
@@ -0,0 +1,543 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
```

Comment:
We don't need to put this in a closure. AFAIK the file itself is a scope (I manually verified it too). None of the variables will be accessible outside of the file unless assigned to global scope via `window.blah = x`